zio-http/src/main/scala/zhttp/http/MimeDB.scala

```diff
@@ -0,0 +1,7787 @@
+package zhttp.http
+
+private[zhttp] trait MimeDB {
```

Comment:
We should be able to set and get content type via MimeDB in `Headers`, `Request` and `Response`